File: result/translate/chatgpt/python2java/java/PROGRAM_TO_CHECK_IF_A_GIVEN_NUMBER_IS_LUCKY_ALL_DIGITS_ARE_DIFFERENT.java
public static boolean f_gold(int n) {
    int[] ar = new int[10];
    while (n > 0) {
        int digit = (int)Math.floor(n % 10);
        if (ar[digit] != 0) {
            return false;
        }
        ar[digit] = 1;
        n = (int)(n / 10);
    }
    return true;
}

File: dataset/python2java/python/PROGRAM_TO_CHECK_IF_A_GIVEN_NUMBER_IS_LUCKY_ALL_DIGITS_ARE_DIFFERENT.py
def f_gold ( n ) :
    ar = [ 0 ] * 10
    while ( n > 0 ) :
        digit = math.floor ( n % 10 )
        if ( ar [ digit ] ) :
            return False
        ar [ digit ] = 1
        n = int ( n / 10 )
    return True